client/network-gossip/src/state_machine.rs

```diff
@@ -528,42 +579,9 @@ mod tests {
 
 	#[test]
 	fn peer_is_removed_on_disconnect() {
-		struct TestNetwork;
```

Comment:
As this is useful for multiple tests, this has been moved one scope up (see above).